### PR TITLE
Add filter for resource groups

### PR DESCRIPTION
**Pull request by TomGudman**
Simple change to be able to query resource groups (aka tags)

### CODE REVIEW

**Comment by harph** [comment]
Hey @dmpaulrohwer can you take care of this?

File: ice_pick/filters/group_by.py [hunk 1]
```diff
@@ -27,3 +27,4 @@
 REGION = 'Region'
 USAGE_TYPE = 'UsageType'
 APPLICATION_GROUP = 'ApplicationGroup'
+RESOURCE_GROUP='ResourceGroup'
```
Comment (harph) [inline ice_pick/filters/group_by.py]: Missing spaces between `=` and

Comment (nstransky) [inline ice_pick/filters/group_by.py]: Hey guys ! Can you take me off this list / repo? Thanks!

**Comment by forrestkirk** [comment]
Hey, can I please be taken off too? Thx!

**Comment by harph** [comment]
@nstransky @forrestkirk just ignore or unwatch the repo

**Pull request merged**